db/migrate/20170406161501_create_forms.rb

```diff
@@ -0,0 +1,12 @@
+class CreateForms < ActiveRecord::Migration[5.0]
+  def change
+    create_table :forms do |t|
+      t.string :description, null: false
+      t.string :url, null: false
+      t.boolean :active, null: false, default: false
+      t.string :created_by, null: false
```

Comment:
What will this field represent? A name of someone who created the form? If so, I'd prefer this to be a foreign key to their record in the database. It adds flexibility later on if you need to grab more information. If you do that, call it user_id instead of created_by.